server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1000,20 +1000,22 @@ private SortedMap<TServerInstance,TabletServerStatus> createTServerStatusView(
         final Map<String,TableInfo> newTableMap =
             new HashMap<>(dl == DataLevel.USER ? oldTableMap.size() : 1);
         if (dl == DataLevel.ROOT) {
-          if (oldTableMap.containsKey(RootTable.NAME)) {
-            newTableMap.put(RootTable.NAME, oldTableMap.get(RootTable.NAME));
+          if (oldTableMap.containsKey(RootTable.ID.canonical())) {
```

Comment:
Why not make it a `Map<TableId,TableInfo>` instead of `Map<String,TableInfo>`?

Reply:
> Why not make it a Map<TableId,TableInfo> instead of Map<String,TableInfo>?

The code is currently manipulating a Thrift object which can not use the TableId type.  A larger refactor would be needed to make that change, could open in issue to look into that.